index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header">
+        <span class="logo"> <img src="" alt="My Logo"></span> 
+        <span class="hamburger"><img src="./img/hamburger.png" alt=""></span>
+    </div>
+    <main class="main">
+        <p class="primary-text">I’m Nshanji. <br>
+            Glad to see you!</p>
+        <p class="content">I’m a software developer! I can help you build a product ,
+                 feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <p class="social">LET'S CONNECT</p>    
+        <div class="social-media-icons">
+            <a href="#"><img src="./img/twitter-icon.svg" alt=""></a>
+            <a href="#"><img src="./img/Linkedin-icon.svg" alt=""></a>
+            <a href="#"><img src="./img/Matador-icon.svg" alt=""></a>
+            <a href="#"><img src="./img/github_icon.svg" alt=""></a>
+            <a href="#"><img src="./img/victory-icon.svg" alt=""></a>
+        </div>     
```

Comment:
- [ ] Your social media icons should be inside the `ul` tap and each icon inside a `li` tag. Kindly fix that.😊